etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.meta.yml

```diff
@@ -0,0 +1,153 @@
+dataset:
+  sources:
+    -
+      name: Penn World Table 10.0
+      published_by: Feenstra, R. C., Inklaar, R. and Timmer, M.P. (2015), "The Next Generation of the Penn World Table". American Economic Review, 105(10), 3150-3182
+
+tables:
+  penn_world_table:
+    variables:
+      rgdpe:
+        title: Expenditure-side real GDP at chained PPPs
+        unit: Million international-$ at 2017 prices
+      rgdpo:
+        title: Output-side real GDP at chained PPPs
+        unit: Million international-$ at 2017 prices
+      pop:
+        title: Population
+        unit: Million people
+      emp:
+        title: Number of persons engaged
+        unit: Million people
+      avh:
+        title: Average annual hours worked by persons engaged
+        unit: 
+      hc:
+        title: Human capital index, based on years of schooling and returns to education; see Human capital in PWT9.
+        unit:   
```

Comment:
I'd be interested to know why you decided to fill out some variable units and not others. Are you ignoring the units of variables that you are not planning to use in garden?

Reply:
In this case it was just because I started doing it and then I stopped because of the number of variables, I might finish this, but for sure it would be more complete for `garden`. Do you have a list of the fields available for metadata?

Reply:
(Speaking of metadata and tedious variables enumeration - have you tried `poetry run etl-metadata-export data/meadow/ggdc/2022-11-28/penn_world_table -o etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.meta.yml` which is mentioned in walkthrough meadow? It generates `*.meta.yml` file with pre-filled variable names)

Reply:
If you don't think you can complete the metadata in meadow, I'd suggest you remove it from meadow and do it in garden (filling out all necessary fields, for which you can use what Mojmir just suggested).
That's what I told to you yesterday: In an ideal world, we should have metadata in both meadow and garden, but in practice we only use the one in garden.
Regarding a list of fields, the easiest is to see other examples of other garden steps, for example [this one](https://github.com/owid/etl/blob/master/etl/steps/data/garden/eia/2022-07-27/energy_consumption.meta.yml).

Reply:
I think I missed that. Thanks, @Marigold!